app/src/main/java/org/mozilla/fenix/browser/BrowserFragment.kt

```diff
@@ -86,6 +87,16 @@ class BrowserFragment : Fragment() {
             engineView
         )
 
+        val findInPageIntegration = FindInPageIntegration(requireComponents.core.sessionManager, findInPageView)
+        val toolbarIntegration = ToolbarIntegration(
```

Comment:
This doesn't look right. There's a `ToolbarIntegration` on line 112 in the `lifecycle.addObservers`. If we're adding it here, we should put this reference in the addObservers call below and remove the inlined one.